src/models/Card/Card.ts

```diff
@@ -14,7 +14,37 @@ import { ICardDetails } from './ICard';
  * Extends Utils class which has bunch of common methods.
  */
 class Card extends Utils {
+  private static DISABLED_ATTRIBUTE: string = 'disabled';
+  private static ERROR_CLASS: string = 'error';
   private static NOT_FLIPPED_CARDS: string[] = [CARD_TYPES.AMEX];
+
+  /**
+   * Clears validation message and error classes (input field, error message).
+   * @private
+   */
+  private static _clearFieldValidationData(inputId: string, messageId: string) {
+    document.getElementById(inputId).classList.remove(Card.ERROR_CLASS);
+    document.getElementById(messageId).textContent = '';
+  }
+
+  /**
+   * Set disable attribute to given input.
+   * @param inputId
+   * @private
+   */
+  private static _disableInput(inputId: string) {
```

Comment:
I've noticed coveralls is complaining on this branch, I guess more unit tests are required to test more of this functionality

Reply:
true, I'll check which lines are not covered